scripts/stats-aggregator/overall.ts

```diff
@@ -0,0 +1,246 @@
+import commandLineArgs from "command-line-args";
+import commandLineUsage from "command-line-usage";
+
+import { BlockNumberRange, prisma } from "@blobscan/db";
+
+import { deleteOptionDef, helpOptionDef } from "./common";
+
+const BEACON_NODE_ENDPOINT = process.env.BEACON_NODE_ENDPOINT;
+const DEFAULT_UNPROCESSED_BLOCKS_BATCH_SIZE = 1_000_000;
```

Comment:
isn't it too big for being the default value?

Reply:
It's' an arbitrary value, still don't know what's an appropriate value for this operation.

Let's reduce it to 100k blocks